lib/Map/CesiumTileLayer.ts

```diff
@@ -118,15 +121,22 @@ export default class CesiumTileLayer extends L.TileLayer {
     let clipPositionWithinMap;
     let clipX;
 
-    const map = this._map;
-    const size = map.getSize();
-    const nw = map.containerPointToLayerPoint([0, 0]);
-    const se = map.containerPointToLayerPoint(size);
-    clipPositionWithinMap = size.x * this.splitPosition;
-    clipX = Math.round(nw.x + clipPositionWithinMap);
-    clipLeft = "rect(" + [nw.y, clipX, se.y, nw.x].join("px,") + "px)";
-    clipRight = "rect(" + [nw.y, se.x, se.y, clipX].join("px,") + "px)";
-
+    if (this.leaflet.size && this.leaflet.nw && this.leaflet.se) {
+      clipPositionWithinMap = this.leaflet.size.x * this.splitPosition;
+      clipX = Math.round(this.leaflet.nw.x + clipPositionWithinMap);
+      clipLeft =
+        "rect(" +
+        [this.leaflet.nw.y, clipX, this.leaflet.se.y, this.leaflet.nw.x].join(
+          "px,"
+        ) +
+        "px)";
+      clipRight =
+        "rect(" +
+        [this.leaflet.nw.y, this.leaflet.se.x, this.leaflet.se.y, clipX].join(
+          "px,"
+        ) +
+        "px)";
+    }
```

Comment:
These are identical, but `size` `nw` `se` are now calculated in `this.leaflet` - as they are now reactive